examples/Finding_Local_Peaks_in_Solar_Data.py

```diff
@@ -0,0 +1,85 @@
+"""
+=================================
+Finding Local Peaks in Solar Data
+=================================
+
+How to find regions of local maxima
+"""
+import astropy.units as u
+from scipy.signal import argrelmax
+import matplotlib.pyplot as plt
+from mpl_toolkits import mplot3d
+from mpl_toolkits.mplot3d import Axes3D
+import numpy as np
+
+import sunpy.map
+from sunpy.data.sample import AIA_193_IMAGE
+
+###############################################################################
+# We will first create a Map using some sample data and display it.
+
+aiamap = sunpy.map.Map(AIA_193_IMAGE)
+plt.figure()
+aiamap.plot()
+plt.colorbar()
+plt.show()
```

Comment:
I think for the other gallery examples we only have one show at the end, so all the plots pop up at the same time.